modules/beta-autopilot-private-cluster/variables.tf

```diff
@@ -524,6 +524,12 @@ variable "enable_l4_ilb_subsetting" {
   default     = false
 }
 
+variable "enable_gcfs" {
+  type        = bool
+  description = "(Beta) Enable image streaming on cluster level."
+  default     = null
```

Comment:
With this approach do we still need ignore_changes or can we revert that?

Reply:
I think it should be fine to revert it now as it doesn't address the desired issue anything.